src/org/parosproxy/paros/extension/history/PopupMenuExportResponse.java

```diff
@@ -66,11 +67,6 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
                     return;
         	    }
 
-                if (hrefs.size() > 1) {
-                    extension.getView().showWarningDialog(Constant.messages.getString("history.export.response.single.warning"));	// ZAP: i18n
-                    return;
-                }
-
                 HttpMessage msg = null;

```

Comment:
The following message validations should be removed as well? (We should validate all or none?)

Reply:
Best to remove or migrate inside loop?

Reply:
I think it's fine to remove them, but we could read the message inside a try/catch block to log warning and continue though(?)